apps/api/src/applicant/external-api.controller.ts

```diff
@@ -84,4 +87,16 @@ export class ExternalAPIController {
   async getLatestSuccessfulSync(): Promise<SyncApplicantsAudit[]> {
     return this.externalAPIService.getLatestSuccessfulSync();
   }
+
+  @ApiOperation({
+    summary: `Sync Users to ATS`,
+  })
+  @UseInterceptors(ClassSerializerInterceptor)
+  @HttpCode(HttpStatus.OK)
+  @Get('/sync-users')
```

Comment:
This naming sync might hint that it will update data.
How about path as /users and method as getUsers